manifests/role/tools_lib.pp

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# stand up apache server w/ dependencies for confluence and apache
+#
+# vhost name is: tools.lib.umich.edu
+#
+#
+# @example
+#   include nebula::role::tools_lib
+class nebula::role::tools_lib (
+  String $domain = 'tools.lib.umich.edu',
+) {
+
+  include nebula::role::aws
+
+  $http_files = lookup('nebula::http_files')
+
+  class { 'nebula::profile::tools_lib::apache':
+    servername => $domain
+  }
+  include nebula::profile::tools_lib::postgres
+
+  include nebula::profile::tools_lib::jdk
+
+  # jira/confluence font requirements
+  package { 'fonts-dejavu-core': }
+  package { 'fontconfig': }
+
+  class { 'nebula::profile::tools_lib::confluence':
```

Comment:
since you're passing the domain here, the profiles don't need a default

Reply:
removed